### PR TITLE
Add test plan for QUARKUS-4120

### PR DESCRIPTION
**Pull request by mocenas**
### Links

JIRA: https://issues.redhat.com/browse/QUARKUS-4120

Quarkus documentation:

### Reminder for considerable topics

 - [X] Make sure you have considered the following areas when preparing the test plan:

   - Logging
   - Tracing
   - Metrics
   - Security
   - OpenAPI
   - Data sources
   - Frontend
   - Qute

### CODE REVIEW

File: QUARKUS-4120.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ - Forcing kafka to send messages to dead-letter-queue.
+ - Confirm that these messages are counter by the micrometer.
+
+## Automated test development
+Quarkus test suite already contains tests for [kafka and micrometer](https://github.com/quarkus-qe/quarkus-test-suite/tree/main/monitoring/micrometer-prometheus-kafka).
+These will be enhanced to support testing of this issue.
+This will require new test class, which will lead to new application build during testing
+
+## Impact on resources
+Additional ~1.5 minutes of test execution time.
```
Comment (fedinskiy) [inline QUARKUS-4120.md:18]: What about native/Openshift?

**Review by fedinskiy — CHANGES_REQUESTED** [review]

File: QUARKUS-4120.md (hunk 1 above)
Comment (mocenas) [reply QUARKUS-4120.md]: I've added time on native. The actual test is not run on OCP as IMHO it is not necessary as this is mainly bug fix.

**Review by fedinskiy — APPROVED** [review]

**Review by fedinskiy — APPROVED** [review]
But put it all into the single commit, please

**Comment by fedinskiy** [comment]
We can, but it requires a manual editing on the part of reviewer, which is something this particular reviewer would like to avoid

**Pull request merged**